Dockerfile

```diff
@@ -79,3 +61,9 @@ RUN pip3 install \
   beautifulsoup4==4.7.1 \
   lxml==4.3.1 \
   pycodestyle==2.5.0
+
+# Install ruby deps with bundler to make things more standard for Ruby folks.
+RUN gem install bundler:2.0.1
```

Comment:
If you don't specify a version, RubyGems will install the latest stable version of a gem. I would recommend using the "pessimistic version constraint" or "twiddle wakka" (`~`) to specify you want the latest version in the 2 series of Bundler. Unless, of course, you specifically need to use `2.0.1`.
With the twiddle waka, you automatically get the latest version of Bundler 2.

Reply:
I want to keep the generated docker image as consistent as possible. A lot of that parnoia comes from my experience with Java but I rather like it. I'm happy to leave a comment at the top of the dockerfile about how the version numbers aren't explicit.

Reply:
You and future maintainers of this file will just have to make sure to update this line if there are newer versions of `Bundler` you'd like to use.